base/accumulate.jl

```diff
@@ -0,0 +1,450 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+"""
+    Accumulate(op[, v0], iter)
+
+An iterator which cumulatively applies the binary operation `op` to the iterator `iter`, in a similar manner
+to [`foldl`](@ref) but returning intermediate values.
```

Comment:
The `accumulate` function would be a more natural thing to reference here, since this is just a lazy (and potentially less accurate for things like `+`, since it doesn't do pairwise summation) `accumulate`.